libs/platform/src/lib/approval-flow/approval-flow.component.ts

```diff
@@ -90,9 +91,24 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
     /** Text label for watchers list */
     @Input() watchersLabel = 'Watchers';
 
+    /** Enables or disables ability to add parallel nodes */
+    @Input() allowAddParallelNodes = true;
+
+    /** Disables save button, save button is enabled by default */
+    @Input() disableSaveButton = false;
```

Comment:
I don't see a request for disabling/enabling the save or exit button in the issue. Where this request come from?

Reply:
@platon-rov request is coming from @835291 , we had discussion meeting around approval flow